misk-hibernate/src/main/kotlin/misk/hibernate/migrate/BulkShardMigrator.kt

```diff
@@ -389,6 +424,14 @@ class BulkShardMigrator<R : DbRoot<R>, C : DbChild<R, C>> private constructor(
       insert.setObject(parameterIndex, value)
       return 1
     }
+
+    open fun bindUpdate(update: PreparedStatement, parameterIndex: Int): Int {
+      return 0
```

Comment:
Should this be abstract or throw `UnsupportedOperationException`?

Reply:
It was used as default behavior, used by ```NowMutation```. Although thinking more about it maybe is better to define it as abstract and force all mutations to implement it.

Reply:
Done